xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/BooksCatalog.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<catalog>
+	<book id="1">
+		<author>King, Stephen</author>
+		<title>IT</title>
+		<genre>Horror</genre>
+		<price>40.00</price>
+	</book>
+	<book id="2">
+		<author>Assis, Machado De</author>
+		<title>Dom Casmurro</title>
+		<genre>Romance</genre>
+		<price>50.00</price>

```

Comment:
This is a duplicate of the other BooksCatalog.xml, we could put it in a resource folder and share it between the projects. This ensures that they both are dealing with the same data and avoids issues like updating one and not the other.